src/interface/src/app/plan/create-scenarios/goal-overlay/goal-overlay.component.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../../../styles/colors";
+@import "../../../../styles/mixins";
+
+:host {
+  @include public-sans();
+  display: block;
+  max-width: 540px;
+  position: absolute;
+  top: 110px;
+  left: 0;
+  z-index: 200;
+  margin: 0 8px;
+
+  @media screen and (max-width: 800px) {
```

Comment:
I've been so tempted to add media queries, so I fully support this.
(At some point -- not in this PR -- maybe we could chat about mixins for breakpoints...)

Reply:
ha sure thing, these ones are  mostly so the modal doesn't completely break the ui when it runs out of place....